code/API_definitions/BYON-Notification-Channel.yaml

```diff
@@ -200,39 +206,89 @@ components:
           type: string
           description: (Optional) Secret used for authentication with the webhook.
           example: "your_secret_key"
+          
+    WebhookData:
+      properties:
+        webhook:
+          $ref: '#/components/schemas/Webhook'
+        expiresAt:
+            type: string
+            format: date-time
+            description: Time indicating when the channel will expire if it is not renewed.
+            example: "2024-02-29T12:00:00Z"
+
 
     WebSocketsData:
       properties:
         channelURL:
           type: string
-          description: "Specified by the server. Contains the URL used to open a WebSockets connection to receive event notification. The channelURL SHALL NOT be included in POST request to create the Notification Channel resource, but MUST be included in the response to the channel creation and any HTTP method that returns an entity body."
-
+          description: >-
+            Specified by the server. Contains the URL used to open a WebSockets
+            connection to receive event notification. The channelURL SHALL NOT
+            be included in POST request to create the Notification Channel
+            resource, but MUST be included in the response to the channel
+            creation and any HTTP method that returns an entity body.
     PNSChannelData:
       description: This specifies the properties of PNS channel the client wish to use
       properties:
         registrationToken:
           type: string
-          description: |
-            The client shall include this in the request & this provides the token assigned by the notification service provider for the device (e.g. APNS or FCM).
-            How client obtain this token is outside the scope of this API specification.
+          description: >-
+            The client shall include this in the request & this provides the
+            token assigned by the notification service provider for the device
+            (e.g. APNS or FCM).
 
+            How client obtain this token is outside the scope of this API
+            specification.
     NotificationChannel:
       required:
-      - channelType
-      - channelData
+        - channelType
+        - channelData
       properties:
         channelType:
           $ref: '#/components/schemas/ChannelType'
         channelData:
           oneOf:
-          - $ref: '#/components/schemas/WebSocketsData'
-          - $ref: '#/components/schemas/PNSChannelData'
-          - $ref: '#/components/schemas/WebhookData'
+            - $ref: '#/components/schemas/WebhookData'
```

Comment:
@rartych in your previous version you did not include WebHook as a notification Channel, Any reason?

Reply:
I have used files from https://github.com/camaraproject/WebRTC/pull/37 (https://github.com/JavierVillaLabarra/WebRTC/tree/main) and corrected the linting issue . I see this branch is a few commits behind main repo.

Reply:
No further problem, I used a file-comparison tool:
- [Notification Channel](https://www.diffchecker.com/52UV2Mf9/)
- [Call Handling](https://www.diffchecker.com/L36AYRr9/)
- [RACM](https://www.diffchecker.com/BQuzqOuv/)